backend/auth/models.go

```diff
@@ -0,0 +1,16 @@
+package auth
+
+import (
+	"gorm.io/gorm"
+	"time"
+)
+
+type Users struct {
+	gorm.Model
+	Name      string
+	Email     string
+	GithubId  string
```

Comment:
We should add github_token on the model

We will use that to fetch his github profile